src/main/java/org/sagebionetworks/web/client/SynapseJavascriptClient.java

```diff
@@ -1534,16 +1533,20 @@ public void initSession(String token) {
 	}
 
 	/**
-	 * call to ask the server to set the session cookie
+	 * call to ask the server to set the access token cookie
 	 * 
 	 * @param token
 	 * @param callback
 	 */
 	public void initSession(String token, AsyncCallback<Void> callback) {
-		Session s = new Session();
-		s.setSessionToken(token);
-		String url = jsniUtils.getSessionCookieUrl();
-		doPost(url, s, OBJECT_TYPE.None, false, callback);
+		AccessTokenWrapper t = new AccessTokenWrapper();
+		t.setToken(token);
+		String url = jsniUtils.getAccessTokenCookieUrl();
+		doPost(url, t, OBJECT_TYPE.None, false, callback);
+	}
+	
+	public FluentFuture<String> getAccessToken() {
```

Comment:
Wrote this as a FluentFuture because I thought I would want to run these two calls in parallel (getting the access token cookie value and the current user profile), but then realized the getMyProfile call depends on the access token (duh).